src/frontend/src/lib/canisters/signer.canister.ts

```diff
@@ -114,7 +114,7 @@ export class SignerCanister extends Canister<SignerService> {
 		);
 
 		if ('Err' in response) {
-			throw mapSignerCanisterBtcError(response.Err);
+			throw mapSignerCanisterSendBtcError(response.Err);
```

Comment:
There is a new variant that needs to be handled, hence a different converter.

Reply:
Yeah, exactly.

Reply:
That must be because before it was using the same error enum as another endpoint. Candid maybe reused two errors instead of already creating both.

Thanks @bitdivine !